src/modules/MouseUtils/FindMyMouse/FindMyMouse.cpp

```diff
@@ -593,7 +593,14 @@ bool SuperSonar<D>::IsForegroundAppExcluded()
     {
         auto processPath = get_process_path(foregroundApp);
         CharUpperBuffW(processPath.data(), static_cast<DWORD>(processPath.length()));
-        return find_app_name_in_path(processPath, m_excludedApps);
+        bool res = find_app_name_in_path(processPath, m_excludedApps);
+
+        if (res == false)
+        {
+            res = check_excluded_app_with_title(foregroundApp, processPath, m_excludedApps);
+        }
```

Comment:
I believe it makes sense to create a function that calls `find_app_name_in_path` and `check_excluded_app_with_title`, as everywhere they're called together. It will help to keep the code clean and avoid copy-pasting.